tests/post_training/conftest.py

```diff
@@ -129,7 +157,10 @@ def accept_pipeline_type(cls, pipeline_type: PipelineType) -> bool:
 
     @classmethod
     @TableColumn.assign_default_value
+    @TableColumn.na_msg
     def get_value(cls, info: Dict[PipelineType, RunInfo], target_pipeline_type: PipelineType) -> str:
+        if info[target_pipeline_type].top_1 is None or info[PipelineType.FP32].top_1 is None:
+            return None
```

Comment:
It will return `None` or `NOT_AVAILABLE_MESSAGE`?

Reply:
What about question? how `return None` can return `NOT_AVAILABLE_MESSAGE`? Don't understand you.